lib/checks/visibility/hidden-content.js

```diff
@@ -1,6 +1,8 @@
 let styles = window.getComputedStyle(node);
+let nodeName = node.nodeName.toUpperCase();
+let skipOver = ['HEAD', 'TEMPLATE', 'SCRIPT'];
```

Comment:
This PR wasn't the one that made it in. Style is in the list, but noscript is. We can remove it later.